src/src/WebServer/JSON.cpp

```diff
@@ -462,7 +462,10 @@ void handle_json()
           stream_next_json_object_value(F("ValueNumber"), x + 1);
           stream_next_json_object_value(F("Name"),        Cache.getTaskDeviceValueName(TaskIndex, x));
           stream_next_json_object_value(F("NrDecimals"),  nrDecimals);
-          stream_last_json_object_value(F("Value"), value);
+          stream_next_json_object_value(F("Value"), value);
+//#if FEATURE_easyfetch

```

Comment:
There should be something like `if (device.easyfetchOption)` here...

Reply:
So how would i make the `device.easyfetchOption` optional in the plugin by selecting easyfetch as the "Output Data Type"?  I want this field only to appear for this specific type..

Reply:
Just checking for it to be not-empty then add it would work, no other plugins allow to enter anything there 🤷‍♂️

Reply:
Yup... probably the best way. Let's see what the other guy has to say about it...